### PR TITLE
feat: SIGINT-996 Add support for user to pass triage flag in CI bridge

### PR DESCRIPTION
**Pull request by kiransynopsys**
1.user can pass bridge_polaris_triage to set triage
2. removed redundant code(polaris assessement type)

### CODE REVIEW

File: src/synopsys-action/tools-parameter.ts [hunk 1]
```diff
@@ -32,19 +32,16 @@ export class SynopsysToolsParameter {
...
-        // converting provided assessmentTypes to uppercase
-        const assessmentTypes = assessmentTypesInput.toUpperCase().split(',')
-        for (const assessmentType of assessmentTypes) {
-          const regEx = new RegExp('^[a-zA-Z]+$')
-          if (assessmentType.trim().length > 0 && regEx.test(assessmentType.trim())) {
-            assessmentTypeArray.push(assessmentType.trim())
-          }
+    if (inputs.POLARIS_ASSESSMENT_TYPES) {
+      // converting provided assessmentTypes to uppercase
+      const assessmentTypes = inputs.POLARIS_ASSESSMENT_TYPES.toUpperCase().split(',')
```
Comment (sig-tithi) [inline src/synopsys-action/tools-parameter.ts:37]: isn't it single value field? why are we accepting multiple values?

File: action.yml [hunk 2]
```diff
@@ -50,6 +50,9 @@ inputs:
   polaris_serverUrl:
     description: 'Polaris Server URL'
     required: false
+  bridge_polaris_triage:
+    description: 'Polaris Triage Option'
```
Comment (sig-tithi) [inline action.yml:54]: Option is not needed

Comment (sig-tithi) [reply action.yml:54]: Also can you take care of contract test and mock bridge changes ?

Comment (kiransynopsys) [reply action.yml:54]: @sig-tithi :  Can I add it as Polaris Triage Value or just Polaris Triage

**Review by sig-tithi — APPROVED** [review]

File: action.yml (hunk 2 above)
Comment (kiransynopsys) [reply action.yml]: updated

File: src/synopsys-action/input-data/polaris.ts [hunk 3]
```diff
@@ -1,13 +1,9 @@
...
-  SCA = 'SCA',
-  SAST = 'SAST'
-}
-
 export interface Polaris {
   polaris: PolarisData
 }
 
 export interface PolarisData {
+  triage?: string
```
Comment (sig-tithi) [inline src/synopsys-action/input-data/polaris.ts:6]: any reason we added ?

**Review by sig-tithi — APPROVED** [review]

File: src/synopsys-action/input-data/polaris.ts (hunk 3 above)
Comment (kiransynopsys) [reply src/synopsys-action/input-data/polaris.ts:6]: It is as optional field

**Review by lsynopsys — APPROVED** [review]

**Pull request merged**